pkg/framework/aries/framework.go

```diff
@@ -43,6 +44,8 @@ type Aries struct {
 	inboundTransport       transport.InboundTransport
 	kmsCreator             api.KMSCreator
 	kms                    api.CloseableKMS
+	cryptoCreator          crypto.Creator
```

Comment:
Why is the framework using an interface defined in API rather than SPI? This doesn't make sense.

Reply:
the framework uses api.Crypto to invoke crypto operations.. and can be injected with SPI.Crypto to build the instance used by the framework

Reply:
I will flip the dependency type around, this should be an SPI crypto type, not `common/api/crypto`

Reply:
done